### PR TITLE
1243 comboinput placeholder translated

### PR DESCRIPTION
**Pull request by mvniekerk**
The combobox's placeholder was not translated.
No it is.

### CODE REVIEW

File: app/modules/forms/forms.tsx [hunk 1]
```diff
@@ -156,6 +156,7 @@ export default class Forms extends React.Component<{}, IFormsState> {
             onMadeDirty={onDirty}
             validation={[isSmart]}
             helpMessage="Try to select a smart animal"
```
Comment (melissapalmer) [inline app/modules/forms/forms.tsx]: If we translating.. should this help message not also be translated?

File: i18n/de/forms.json [hunk 2]
```diff
@@ -4,6 +4,9 @@
       "yes": "Ja",
       "no": "Nein",
       "maybe": "Könnte sein"
+    },
+    "comboInput": {
+      "placeholder": "Select a smart animal"
```
Comment (melissapalmer) [inline i18n/de/forms.json]: This is the German file... text needs to be German.

File: app/modules/forms/forms.tsx (hunk 1 above)
Comment (mvniekerk) [reply app/modules/forms/forms.tsx]: The fields are made that you can decide if you want a raw string or a translation key

File: i18n/de/forms.json (hunk 2 above)
Comment (mvniekerk) [reply i18n/de/forms.json]: d09c18a  updated

**Pull request merged**